### PR TITLE
Interaction Recording Helper Function

### PR DESCRIPTION
**Pull request by dprevoznik**
Added interaction recording helper function. Also, implemented the helper function on left and right carousel button interaction points.

### CODE REVIEW

File: src/components/productDetails/card.jsx [hunk 1]
```diff
@@ -38,13 +39,21 @@ const Card = ({
...
               setShowModal(true);
+              recordInteraction(
+                `button.button.is-small.actionButton.product-${product.id}.${listName}`,
+                "related-items-comparison"
+              );
             } else {
               onClickButton(listName, product.id);
+              recordInteraction(
+                `button.button.is-small.actionButton.product-${product.id}.${listName}`,
+                "related-items-comparison"
```
Comment (mayannechess) [inline src/components/productDetails/card.jsx:55]: We talked about level of specificity, making sure we all have at least e.g. button & id

File: src/interactionsHelper.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+};
+
+export default recordInteraction;
+
+// EXAMPLE OBJECT SENT
+// var exampleData = {
+//   element: "div.button.related-product",
+//   widget: "related-items-comparison",
+//   time: "Wed Apr 01 2020 11:05:20 GMT-0400 (Eastern Daylight Time)"
+// };
```
Comment (iwantmyhatback) [inline src/interactionsHelper.js:28]: This is great! will implement ASAP

**Review by iwantmyhatback — APPROVED** [review]
Sir, I will be using this as-well. Approved

**Review by mayannechess — APPROVED** [review]
This will make the change request much easier to handle. We talked about inspecting elements in devtools to get appropriate selectors to send into the function.

**Pull request merged**